docs/install.md

```diff
@@ -157,3 +157,22 @@ Toolchains allow us to support cross-compilation, e.g. building a linux binary f
 So if for example you want to build a docker image from a non-linux platform you would run `bazel build --platforms=@build_bazel_rules_nodejs//toolchains/node:linux_amd64 //app`, which will ensure that the linux nodejs binary is downloaded and provided to the nodejs_binary target.
 
 Note: The toolchain currently only provides a platform-specific nodejs binary. Any native modules will still be fetched/built, by npm/yarn, for your host platform, so they will not work on the target platform. Support for cross-compilation with native dependencies will follow.
+
+Because these rules use the target platform to decide which node binary to use, you will run into trouble if you are trying to invoke these rules as part of a cross-compilation
```

Comment:
Perhaps we should add something to /examples for this?

Reply:
filed https://github.com/bazelbuild/rules_nodejs/issues/2606 as a reminder so I can merge this